lib/completions.js

```diff
@@ -0,0 +1,3 @@
+const {PipelineCompletion} = require('cli-engine-heroku/lib/completions')
```

Comment:
Out of curiosity, why creating a separate file only for exporting `require('cli-engine-heroku/lib/completions')`? Is it because we're still iterating on the place of where the `completions` library will live?

Reply:
Correct, it may move before GA, but also, I wanted to give you all a single place to override the functionality if you so choose.